src/Hero6.UserInterface.SierraVga/ViewModel/TextBoxViewModel.cs

```diff
@@ -49,31 +50,28 @@ public void Show(string input)
             Size size = FontManager.DefaultFont.MeasureString(input, 1, 1);
             int screenWidth = this.nativeWidth * (int)this.scale.X;
             int screenHeight = this.nativeHeight * (int)this.scale.Y;
-            int horizontalEmptySpace = screenWidth / 8;
-            int rowSize = screenWidth - (horizontalEmptySpace * 2);
-            int rowCount = (int)Math.Ceiling(size.Width / rowSize);
+            int screenColumnWidth = screenWidth / 8;
+            int displayAreaColumnWidth = screenWidth - (screenColumnWidth * 2);
+
+            Padding padding = new Padding(1f);
```

Comment:
This looks like a constant actually. Maybe consider making it a private readonly field of the class.